### PR TITLE
[Reopen]Append more test case

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/parser/test/ParserTest.cpp [hunk 1]
```diff
@@ -94,6 +94,33 @@ TEST(Parser, Go) {
...
+        GQLParser parser;
+        std::string query = "CREATE SPACE default_space(partition_num=9, replica_factor=3)";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "CREATE SPACE default_space(partition_num=9, replica_factor=3)";
+        auto result = parser.parse(query);
+        ASSERT_FALSE(result.ok());
```
Comment (steppenwolfyuetong) [inline src/parser/test/ParserTest.cpp]: duplicate ?

File: src/parser/test/ParserTest.cpp [hunk 2]
```diff
@@ -94,6 +94,33 @@ TEST(Parser, Go) {
...
+    }
+    {
+        GQLParser parser;
+        std::string query = "CREATE SPACE space_without_options()";
```
Comment (steppenwolfyuetong) [inline src/parser/test/ParserTest.cpp]: not parentheses

Comment (darionyaphet) [reply src/parser/test/ParserTest.cpp]: pending [Allow to create graph space without options](https://github.com/vesoft-inc/nebula/pull/408)

Comment (steppenwolfyuetong) [reply src/parser/test/ParserTest.cpp]: Yeah, you can go to see the grammar about create space in PR #408

**Comment by darionyaphet** [comment]
jenkins go

File: src/graph/test/SchemaTest.cpp [hunk 3]
```diff
@@ -106,7 +106,7 @@ TEST_F(SchemaTest, metaCommunication) {
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
     }
     {
-        // test exist
+        // test tag have exist
```
Comment (dangleptr) [inline src/graph/test/SchemaTest.cpp]: Test the tag has been existed.

File: src/graph/test/SchemaTest.cpp [hunk 4]
```diff
@@ -45,10 +45,10 @@ TEST_F(SchemaTest, metaCommunication) {
...
         ASSERT_TRUE(verifyResult(resp, expected));
     }
-    // test nonexistent space
     {
+        // test nonexistent space
         cpp2::ExecutionResponse resp;
-        std::string query = "USE default_space";
+        std::string query = "USE not_exist_space";
         auto code = client->execute(query, resp);
         ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```
Comment (dangleptr) [inline src/graph/test/SchemaTest.cpp]: Why return succeeded if the space not exist?

File: src/graph/test/SchemaTest.cpp [hunk 5]
```diff
@@ -151,14 +151,21 @@ TEST_F(SchemaTest, metaCommunication) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    {
+        // test tag not exist
+        cpp2::ExecutionResponse resp;
+        std::string query = "DESCRIBE TAG not_exist";
+        auto code = client->execute(query, resp);
+        ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```
Comment (dangleptr) [inline src/graph/test/SchemaTest.cpp]: Why return succeeded?

Comment (darionyaphet) [reply src/graph/test/SchemaTest.cpp]: `ASSERT_NE`

File: src/graph/test/SchemaTest.cpp (hunk 4 above)
Comment (darionyaphet) [reply src/graph/test/SchemaTest.cpp]: using `E_EXECUTION_ERROR` to check return code and avoid `ASSERT_NE`

Comment (dangleptr) [reply src/graph/test/SchemaTest.cpp]: A HA,  my bad.

File: src/graph/test/SchemaTest.cpp [hunk 6]
```diff
@@ -150,16 +126,17 @@ TEST_F(SchemaTest, metaCommunication) {
         ASSERT_TRUE(verifyResult(resp, expected));
     }
     {
+        // Test the tag has been existed.
         cpp2::ExecutionResponse resp;
-        std::string query = "CREATE TAG upper(name string, EMAIL string, "
+        std::string query = "CREATE TAG person(name string, email string, "
```
Comment (laura-ding) [inline src/graph/test/SchemaTest.cpp]: "EMAIL" is to test unreserved keyword

Comment (steppenwolfyuetong) [reply src/graph/test/SchemaTest.cpp]: Here not test if the tag has been existed，it  is to test unreserved keyword.

Comment (darionyaphet) [reply src/graph/test/SchemaTest.cpp]: fixed

File: src/graph/test/SchemaTest.cpp [hunk 7]
```diff
@@ -150,16 +126,17 @@ TEST_F(SchemaTest, metaCommunication) {
...
                             "age int, gender string, row_timestamp timestamp)";
         auto code = client->execute(query, resp);
-        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);
     }
     sleep(FLAGS_load_data_interval_secs + 1);
     {
         cpp2::ExecutionResponse resp;
-        std::string query = "DESCRIBE TAG upper";
+        std::string query = "DESCRIBE TAG person";
```
Comment (steppenwolfyuetong) [inline src/graph/test/SchemaTest.cpp]: From the original version of line 130 to here, is testing unreversed keyword.
Please recover

Comment (darionyaphet) [reply src/graph/test/SchemaTest.cpp]: good ~

**Comment by darionyaphet** [comment]
jenkins go

**Pull request merged**